orchestra/models/core/models.py

```diff
@@ -197,6 +197,7 @@ class Step(StepMixin, models.Model):
     user_interface = JSONField(default={})
     todolist_templates_to_apply = models.ManyToManyField('TodoListTemplate',
                                                          blank=True)
+    completion_ends_project = models.BooleanField(default=False)
```

Comment:
we also have to load this field from version.json (orchestra/workflow/load.py).

you will then be able to test that the load logic works.
* you can do this automatically in orchestra/tests/workflows/test_load.py
* you can do this manually by updating a workflow, running `./manage.py loadallworkflows --force`, and checking the Django admin to make sure the value loaded properly.

Reply:
done!